### ISSUE TITLE
Dockerfile

### ISSUE THREAD

**Issue by misago162** [kishwarshafin/pepper#65]
Hi. 

We have create a singularity container with peeper and when we execute the pipeline we get this error message:

Traceback (most recent call last):
  File "/usr/local/bin/pepper_hp", line 11, in <module>
    load_entry_point('pepper-polish==0.4.1', 'console_scripts', 'pepper_hp')()
  File "/usr/local/lib64/python3.6/site-packages/pepper_hp/pepper_hp.py", line 433, in main
    set_profile)
  File "/usr/local/lib64/python3.6/site-packages/pepper_hp/modules/python/CallVariant.py", line 115, in call_variant
    threads)
  File "/usr/local/lib64/python3.6/site-packages/pepper_hp/modules/python/RunInference.py", line 143, in run_inference
    output_dir)
  File "/usr/local/lib64/python3.6/site-packages/pepper_hp/modules/python/RunInference.py", line 108, in polish_genome_distributed_cpu
    predict_distributed_cpu(image_dir, file_chunks, output_dir, model_path, batch_size, callers, threads_per_caller, num_workers)
  File "/usr/local/lib64/python3.6/site-packages/pepper_hp/modules/python/models/predict_cpu.py", line 139, in predict_distributed_cpu
    num_classes=ImageSizeOptions.TOTAL_LABELS)
  File "/usr/local/lib64/python3.6/site-packages/pepper_hp/modules/python/models/ModelHander.py", line 109, in load_simple_model_for_training
    transducer_model.load_state_dict(new_model_state_dict)
  File "/usr/local/lib64/python3.6/site-packages/torch/nn/modules/module.py", line 830, in load_state_dict
    self.__class__.__name__, "\n\t".join(error_msgs)))
RuntimeError: Error(s) in loading state_dict for TransducerGRU:
	size mismatch for dense1.weight: copying a param with shape torch.Size([15, 256]) from checkpoint, the shape in current model is torch.Size([5, 256]).
	size mismatch for dense1.bias: copying a param with shape torch.Size([15]) from checkpoint, the shape in current model is torch.Size([5]).

I have discover today that exist a docker container with the pepper + deepvariant and I want to check if this container is working fine for us: 

singularity pull docker://kishwars/pepper_deepvariant:r0.4

but in my cluster fails because we are using RedHat 6. The error we get is 'kernel too old'. 

So, I want to ask you if it is possible you share with the the content of the Dockerfile you have use to generate this docker container. My idea is modify this Dockerfile to generate a docker container with Centos7 or Ubuntu16 that are the operative system that works fine in our cluster. 

Thanks a lot.

**Comment by kishwarshafin** [comment]
@misago162 we are going to update the codebase in a week or two with the guppy 5 support. Will it be OK to wait until the next release?

**Comment by misago162** [comment]
@kishwarshafin  OK, we can wait this time, no problem. Do you know if the Dokerfile will be published?. If we have access to the content of the Dockerfile we can try to generate a docker or singularity container to generate a container with ubuntu-16 or centos7 (these are the operative system that are working in our cluster with RH6, a container with a newer operative system fails with the error message 'kernel too old').  Thanks a lot for you help.

**Comment by kishwarshafin** [comment]
Yes, we will for sure release the docker this time. Btw, the current docker is built on ubuntu-16, would you mind giving it a try on ubuntu 16 and see if you are still facing the issue?

**Comment by misago162** [comment]
Hi, I want to ask you if you know when the update will be released with the content of the Dockerfile. As I explained before, I tried to use the Singularity container (singularity pull docker://kishwars/pepper_deepvariant:r0.4) in our cluster (RedHat Red Hat Enterprise Linux Server release 6.7 (Santiago)) but it fails with the error Kernel too Old. This is the reason I would like to get the content of the Dockerfile, to try to adapt this container to be able to work in our cluster (normally we can run containers that are using Ubuntu-16.04 or Centos-7.9) Thanks a lot for the information.
